### DISCUSSION TITLE
how to get text width like ctx.measureText in canvas

### DISCUSSION THREAD

**Discussion by rhinonan** [antvis/X6#1607] [Q&A] [answered]
how to get text width like ctx.measureText in canvas.
如何像ctx.measureText 一样获取一个文字的宽度。
主要是 想画一个矩形，宽度随文字的变化而变化。

**Comment by outian** [comment]
同求，请问现在有解决方案了吗

**Comment by NewByVector** [comment, marked as answer]
https://bl.ocks.org/MSCAU/58bba77cdcae42fc2f44
